scripts/run-integration-tests.sh

```diff
@@ -140,13 +150,16 @@ mkdir -p "$REPORT_DIR"
 mkdir -p "$TEST_CLUSTER_DIR"
 mkdir -p "$TEST_CONFIG_DIR"
 
-if [[ "$PROVISION" == true ]]; then
-    START=$SECONDS
+START=$SECONDS
+if [[ "$PROVISION" == true && "$RUN_KOPS_TEST" == false ]]; then
```

Comment:
I think we need to switch this logic, because if you set `PROVISION=false`, you will get a Kops cluster. 
```bash
if [[ "$PROVISION" == true ]]; then
  up-test-cluster
else if [[ "$RUN_KOPS_TEST" == true ]]; then
  up-kops-cluster
fi
```